spec/services/ccms/payload_generators/entitiy_attributes_generator_spec.rb

```diff
@@ -7,11 +7,27 @@ module PayloadGenerators
       let(:requestor) { double CCMS::Requestors::CaseAddRequestor, submission: submission, ccms_attribute_keys: yaml_keys }
       let(:submission) { double CCMS::Submission, id: '34343434', legal_aid_application: legal_aid_application }
       let(:legal_aid_application) { double LegalAidApplication }
-      let(:entity_name) { :bank_acct }
+      let(:entity_name) { [:bank_acct, 'bank_acct'].sample }
       let(:yaml_keys) { YAML.load_file(Rails.root.join('spec/fixtures/files/ccms_keys/standard_ccms_keys.yml')) }
       let(:xml) { double Nokogiri::XML::Builder }
       let(:options) { {} }
 
+      describe '#call' do
+        context 'non_passported attributes' do
```

Comment:
- [ ] Update this to no longer be non_passported but to standard ccms keys (passported?)

Reply:
- [ ] Test EntityAttributesGenerator.call more thoroughly and not just for exceptions.